guides/image-sources/provisioning-installer-bootdisk-subnet.plantuml

```diff
@@ -0,0 +1,79 @@
+@startuml
+
+!include foreman.pstyle
+!$networkboot = 1
+
+title Installer-based provisioning with subnet boot disk on bare metal
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+participant DHCP
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
+note over Host : Powered off
+
+== Create host in Foreman ==
+
+User -> Foreman : Create host
+User -> Foreman : Assign an MAC address
+Foreman -> Proxy : Create DNS records
+Proxy -> DNS : Forward DNS records
+Proxy -> Foreman : Report status of DNS
+Foreman -> User : Host is created\n(build mode enabled)
+
+User -> Foreman : Request to generate the subnet boot-disk ISO
+group Templates [Boot disk iPXE - host, PXELinux, PXEGrub2]
+    Foreman -> Proxy : Render boot-loader configuration files
+end
+Foreman -> Foreman : Add the boot loader to ISO files
+Foreman -> Foreman : Add the boot-loader configuration files to ISO files
+Foreman -> Foreman : Build the subnet boot-disk ISO
+
+User -> Foreman : Download the subnet boot-disk ISO
+User -> User : Write the boot disk\nto an external storage device
+User -> Host : Configure to boot\nfrom the external storage device
+
+== Boot into OS installer ==
+
+User -> Host : Power on
+Host -> DHCP : Request the reserved IP address
+Host -> Host : Boot from the external storage device
+Host -> Host : Load the boot loader from the boot disk
+Host -> Host : Load boot-loader configuration from the boot disk
```

Comment:
After the step
```
Host -> Host : Load the boot loader from the boot disk
```
There is another step that I missed during initial reviews:
```
Host -> Proxy: Chainload the iPXE configuration for the host
Proxy -> Foreman: Render the iPXE configuration for the host
```

Reply:
@stejskalleos Which template does it render from?

Reply:
Actually, I just need the template kind here.